src/scss/base/_layout.scss

```
@@ -2,6 +2,7 @@
 @use '../settings/variables' as v;
 @use '../settings/include-media' as *;
 @use '../tools/mixins/type';
+@use '../tools/functions/spacing' as spacing;
```

Minor thing, you can leave the `as spacing` part off and it'll use `spacing` as the module name, like does `type` above.

```suggestion
@use '../tools/functions/spacing';
```